### DISCUSSION TITLE
building examples fails with: error: command failed: 'trunk': No such file or directory (os error 2)

### DISCUSSION THREAD

**Discussion by hpepper** [leptos-rs/leptos#2050] [Q&A] [answered]
Hi

OS: Ubuntu 22.04.3 LTS
Architecture: amd64
Rust: 1.76.0-nightly 

Last few lines of the error:

```text
[cargo-make] INFO - cargo make 0.37.4
...
warning: `auto` traits are unstable
...
    Finished dev [unoptimized + debuginfo] target(s) in 6.75s
[cargo-make][1] INFO - Execute Command: "rustup" "run" "nightly" "trunk" "build"
error: command failed: 'trunk': No such file or directory (os error 2)
[cargo-make][1] ERROR - Error while executing command, exit code: 1
[cargo-make][1] WARN - Build Failed.
[cargo-make] ERROR - Error while running duckscript: Source: Unknown Line: 5 - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
```

I have run the following commands:

1. cd leptos/examples
2. git checkout main
3. git up
4. rustup toolchain install nightly
5. rustup target add wasm32-unknown-unknown
6. rustup default nightly
7. cargo install --force cargo-make
8. cargo make ci

Any idea of what I might have missed that I need to do.

  Thanks

**Comment by gbj** [comment, marked as answer]
```
error: command failed: 'trunk': No such file or directory (os error 2)
```
Do you have `trunk` installed?

**Reply by hpepper** [reply]
Thank you very much, I greatly appreciate you help.
I installed trunk: `cargo install --locked trunk`
and it works now.
I had wrongfully assumed the 'trunk' was a conceptual  reference to git branch.
